SVSModel/Simulation.cs

```diff
@@ -199,18 +199,18 @@ public class SimulationType
 
     public class CropNBalanceSummary
     {
-        public double MineralIn;
-        public double CropIn;
-        public double ResidueIn;
-        public double SOMIn;
-        public double FertiliserIn;
+        public double MineralIn { get; }
+        public double CropIn { get; }
+        public double ResidueIn { get; }
+        public double SOMIn { get; }
+        public double FertiliserIn { get; }
```

Comment:
Don't these properties need to have setters on them as well so they can be set by the constructor.

Reply:
The setters are only required if you intend to set values outside of the constructor.
Initially this class had no getters or setters, so the webapp was not able to actually get the values.

Reply:
I didn't know that but it makes sense when you think about it.